src/theme.js

```diff
@@ -1,9 +1,15 @@
 export const theme = {
     color: {
-        white: "#FFFFFF"
+        white: "#FFFFFF",
+        whisper: "#F5F5FA",
+        mystic: "#E4E6F0",
+        waterloo: "#7E839A",
+        woodsmoke: "#18181B",
+        stormGrey: "#74788B",
+        heater: "#BAC7D5",
+        black: "#000000",
     },
     breakpoint: {
-        tablet: "767px",
```

Comment:
Tego breakpointu na tablet całkowicie bym nie usuwał, tylko go zostawił na 1278px, bo nawet jak w innych elementach strony go nie użyjemy, to przyda się chociaż w tym górnym pasku z nawigacją, bo tam jest za dużo elementów, żeby widok zmienił się dopiero przy 767px 😋

Reply:
Ok, to go zostawiamy, już wklejony.